TodoApp/test/TodoApp.Api.Tests/Controllers/ItemsControllerTests.cs

```diff
@@ -54,6 +61,7 @@ public void SetUp()
         [Test]
         public async Task GetAsync_ReturnsAllItems()
         {
+            _repository.GetAllAsync().Returns(_items);
             var actionResult = await _controller.GetAsync();
             var contentResult = await actionResult.ExecuteAsync(CancellationToken.None);
             contentResult.TryGetContentValue(out Item[] value);
```

Comment:
This 3 lines of code repeat quite often in tests; consider introducing some generic private method that would allow you to make this a one-liner (you can, for example, leverage C# 7's deconstruction and tuples syntactic sugar similarly to demonstrated [here](https://blogs.msdn.microsoft.com/seteplia/2017/11/01/dissecting-the-tuples-in-c-7/).